gtsfm/two_view_estimator.py

```diff
@@ -271,7 +279,7 @@ def compute_correspondence_metrics(
         epipolar_distance_threshold,
     )
     inlier_ratio_gt_model = num_inliers_gt_model / corr_idxs_i1i2.shape[0]
-    return num_inliers_gt_model, inlier_ratio_gt_model
+    return num_inliers_gt_model, inlier_ratio_gt_model, v_corr_idxs_inlier_mask_gt
```

Comment:
even here, ideally we only need to return the mask, num inliers are inlier ratio can be computed outside.